test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -2117,6 +2294,69 @@ func checkFailureReason(t *testing.T, err error, expectedReason metav1.StatusRea
 	}
 }
 
+func unmarshalSchema(t *testing.T, schemaJSON []byte) *apiextensionsv1.JSONSchemaProps {
+	var c apiextensionsv1.JSONSchemaProps
+	err := json.Unmarshal(schemaJSON, &c)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return &c
+}
+
+func withCRDParamKind(kind string) *admissionregistrationv1alpha1.ParamKind {
+	return &admissionregistrationv1alpha1.ParamKind{
+		APIVersion: crdGroup + "/v1beta1",
+		Kind:       kind,
+	}
+}
+
+func genCRD(name, kind string) *apiextensionsv1.CustomResourceDefinition {
```

Comment:
Thoughts on using `versionedCustomResourceDefinition()`? It uses an "allow all" schema which would remove the need to include a schema in the tests

Reply:
It also includes multiple apiVersions which is needed for any tests validating behavior with conversion

Reply:
(as well as the crd* constants)

Reply:
(marking as resolved since `genCRD` is removed)